README.rst

```diff
@@ -141,7 +141,7 @@ Links
 * `Web-Chat (IRC) <http://webchat.freenode.net/?randomnick=1&channels=%23borgbackup&uio=MTY9dHJ1ZSY5PXRydWUa8>`_ and
   `Mailing List <https://mail.python.org/mailman/listinfo/borgbackup>`_
 * `License <https://borgbackup.readthedocs.org/en/stable/authors.html#license>`_
-* `Security contact <https://borgbackup.readthedocs.io/en/latest/support.html#security-contact>`_
+* `Security contact <https://borgbackup.readthedocs.io/en/latest/free.html#security-contact>`_
```

Comment:
guess this should stay?
the file in the docs is still named support.html, isn't it?
just the file on the homepage is free.html.

Reply:
That is correct. I searched for all usages of support.html, and those are the only ones still around.

Reply:
I mean the html docs file name is still support.html (so do not change that to free.html).